src/main/java/de/retest/recheck/cli/utils/FilterUtil.java

```diff
@@ -65,4 +65,14 @@ public static Filter loadRecheckIgnore() {
 		}
 		return Filter.FILTER_NOTHING;
 	}
+
+	public static boolean hasValidExcludeOption( final List<String> exclude ) {
+		final List<String> invalidFilters = FilterUtil.getInvalidFilters( exclude );
+		if ( !invalidFilters.isEmpty() ) {
+			FilterUtil.logWarningForInvalidFilters( invalidFilters );
```

Comment:
Please do not qualify static methods if they themselves are within the current class scope.
```suggestion
			logWarningForInvalidFilters( invalidFilters );
```